src/components/Nav/index.js

```diff
@@ -1,13 +1,29 @@
 import { Link } from "react-router-dom";
 import "./nav.css";
 
-function Nav() {
+function Nav(props) {
+  function handleAvailableClick() {
+    props.setClaimedUrl(
+      "https://book-swap-api.dev.io-academy.uk/api/books?claimed=0"
+    );
+  }
+
+  function handleClaimedClick() {
+    props.setClaimedUrl(
+      "https://book-swap-api.dev.io-academy.uk/api/books?claimed=1"
+    );
+  }
+
   return (
     <nav className="navBar">
-      <a href="/">Books Swap</a>
+      <a href="/" className="bookSwap">Books Swap</a>
       <div className="right-links">
-        <Link to="/">Available books</Link>
-        <Link to="/claimed/">Claimed books</Link>
+        <Link to="/" onClick={handleAvailableClick}>
+          Available books
+        </Link>
+        <Link to="/" onClick={handleClaimedClick}>
+          Claimed Book
+        </Link>
```

Comment:
Now it's not possible to link to claimed/unclaimed books. It was better when you had two separate pages / and /claimed